core/src/main/scala/org/http4s/rho/AuthedContext.scala

```diff
@@ -37,7 +39,7 @@ import org.http4s.rho.bits.{FailureResponse, SuccessResponse, TypedHeader}
   *
   * @tparam U authInfo type for this service.
   */
-class AuthedContext[U] {
+class AuthedContext[F[_]: Monad, U] extends FailureResponseOps[F] {
```

Comment:
Should this be `U, F[_]: Monad`?

Reply:
Why?

Reply:
Partial Unification Reasons, but I could be wrong.

Reply:
@ChristopherDavenport what would break? how can we test? @danielkarch his comment is related to a specific case in http4s where the params order had to be changed - afaik it happened in a single place there